src/webgl/p5.RendererGL.js

```diff
@@ -32,77 +33,135 @@ defineStrokeJoinEnum('ROUND', 0);
 defineStrokeJoinEnum('MITER', 1);
 defineStrokeJoinEnum('BEVEL', 2);
 
-const lightingShader = readFileSync(
+this._pInst.shaderCache={};
```

Comment:
I think this is probably the cause, over here, `this` is undefined. If you want it to be initialized on a `RendererGL` instance, I think moving this line to the `RendererGL` constructor would work.